src/selfplay/loop.cc

```diff
@@ -85,6 +85,11 @@ void SelfPlayLoop::CmdStart() {
       std::make_unique<std::thread>([this]() { tournament_->RunBlocking(); });
 }
 
+void SelfPlayLoop::CmdStop() {
+  tournament_->Stop();
+  tournament_->Wait();
```

Comment:
This wait is here so you can issue stop/quit as a pair - and when the app has exited its done.

Reply:
"Nicer" solution would be stop+isready+quit, but selfplay is not real UCI anyway.